camera_calibration_parsers/src/camera_calibration_parsers/__init__.py

```diff
@@ -1,9 +1,40 @@
+# Copyright 2024 Open Source Robotics Foundation, Inc.
+# All rights reserved.
+#
```

Comment:
This doesn't seem exactly right to me.  This file was originally contributed as part of commit 9829b02b48bae412a087e2183b3fd2c94fb16f26 in 2016.

Having no "Copyright" line at the beginning would be the most correct.  Do the linters succeed in that case?

(there are a few other instances of this pattern below)

Reply:
maybe we can exclude these files from the copyright linter

Reply:
friendly ping @clalancette